docs/commands/index.md

```diff
@@ -178,4 +180,14 @@ theme watch --notify=/tmp/theme.update
 
 |**Optional Flags**||
 |`-a`|`--allenvs`| Will run this command for each environment in your config file.
-|`-n`|`--notify` | File path to a file that you want updated on idle.
+|`-n`|`--notify` | Filepath or URL. Filepath is to a file that you want updated on idle. The URL path is where you want a webhook posted to to report on file changes.
```

Comment:
@tanema the longer line here seems to have messed up the layout, perhaps we can add a line break before the last sentence?

<img width="913" alt="Screen Shot 2020-09-02 at 10 26 18 AM" src="https://user-images.githubusercontent.com/13400/91996331-d9547900-ed06-11ea-9e9f-246f022253d8.png">